acceptance/dlt/install-dlt/script

```diff
@@ -0,0 +1,20 @@
+title "install dlt"
+dlt=$(trace errcode $CLI install-dlt -o json | jq -r .symlink_path)
```

Comment:
I don't think calling jq here does anything. I'd recommend directly running the command and asserting on it's stdout output.
```suggestion
trace errcode $CLI install-dlt
```